projects/components/src/datetime-picker/datetime-picker.component.ts

```diff
@@ -53,7 +53,12 @@ export class DatetimePickerComponent implements OnChanges {
   }
 
   public getInputDate(): string {
-    return this.date?.toISOString().slice(0, 10) ?? '';
+    if (!this.date) return '';
```

Comment:
@adisreyaj thanks for picking it up. Could we use `Time()` with `isUTC` set to true? Just trying to keep conversion at one place

Reply:
Will work if all the provided times in the dropdown uses `isUTC: true` and we use `setUTCHours()` instead of `setHours()` to patch the date when time changes.

Reply:
Ok. Can we do that then?

Reply:
Yup. On it!